src/modules/ChannelSettings/components/ModerationPanel/MembersModal.tsx

```diff
@@ -67,123 +67,125 @@ export default function MembersModal({ onCancel }: Props): ReactElement {
           })}
         >
           {
-            members.map((member: Member) => (
-              <UserListItem
-                user={member}
-                key={member.userId}
-                currentUser={currentUser}
-                action={({ parentRef, actionRef }) => (
-                  <>
-                    {channel?.myRole === 'operator' && (
-                      <ContextMenu
-                        menuTrigger={(toggleDropdown) => (
-                          <IconButton
-                            className="sendbird-user-message__more__menu"
-                            width="32px"
-                            height="32px"
-                            onClick={() => {
-                              toggleDropdown();
-                            }}
-                          >
-                            <Icon
-                              width="24px"
-                              height="24px"
-                              type={IconTypes.MORE}
-                              fillColor={IconColors.CONTENT_INVERSE}
-                            />
-                          </IconButton>
-                        )}
-                        menuItems={(closeDropdown) => (
-                          <MenuItems
-                            parentContainRef={parentRef}
-                            parentRef={actionRef} // for catching location(x, y) of MenuItems
-                            closeDropdown={closeDropdown}
-                            openLeft
-                          >
-                            <OperatorMenuItem
-                              channel={channel}
-                              user={member}
-                              disable={currentUser === member.userId}
-                              onChange={(_, member, isOperator) => {
-                                setMembers(members.map(m => {
-                                  if (m.userId === member.userId) {
-                                    return {
-                                      ...member,
-                                      role: isOperator ? Role.OPERATOR : Role.NONE,
-                                    };
-                                  }
-                                  return m;
-                                }));
-                                closeDropdown();
-                              }}
-                              onError={() => {
-                                // FIXME: handle error later
-                                closeDropdown();
+            members.map((member: Member) => {
```

Comment:
can you explain what has changed here?

Reply:
Ah, just
```tsx
() => (<></>)
```
has changed to
```tsx
() => {
  return (<></>)
}
```
This is easier to see log, that's why